demo/index.js

```diff
@@ -109,7 +108,7 @@ const setPreview = query => {
 const convert = () => {
   try {
     const output = jsxslack([jsxEditor.getValue()])
-    const encoded = encodeURIComponent(JSON.stringify(output))
+    const encoded = JSON.stringify(output)
```

Comment:
It seems to vanish `+` in JSON while translating.

```jsx
<Blocks>
  <Section>y=ax^2+b</Section>
</Blocks>
```

<img width="223" src="https://user-images.githubusercontent.com/3993388/67561785-6d090080-f758-11e9-8f58-90100d2dcc65.png">